### PR TITLE
Fix creating a user throws an error 

### PR DESCRIPTION
**Pull request by imf-ali**
## What does this PR do?

Creates user successfully without errors.

## Behaviour
image

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

* https://github.com/appwrite/sdk-for-cli/issues/81

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yes

### CODE REVIEW

File: templates/cli/lib/parser.js.twig [hunk 1]
```diff
@@ -8,7 +8,7 @@ const cliConfig = {
   json: false
 };
 
-const parse = (data) => {
+const parse = (data, child = false) => {
```
Comment (stnguyen90) [inline templates/cli/lib/parser.js.twig:11]: I don't think using child like this is the right approach because it doesn't work for multiple levels deep.

Comment (imf-ali) [reply templates/cli/lib/parser.js.twig]: Fixed

File: templates/cli/lib/parser.js.twig [hunk 2]
```diff
@@ -23,11 +23,11 @@ const parse = (data) => {
                 drawJSON(data[key]);
             }
         } else if (typeof data[key] === 'object') {
-            if (data[key] && data[key].constructor.name === 'BigNumber') {
+            if (data[key] && data[key].constructor?.name === 'BigNumber') {
```
[the remark this answers is not in the document]
Comment (imf-ali) [reply templates/cli/lib/parser.js.twig]: Done

**Comment by imf-ali** [comment]
I have addressed the comments. Kindly have a look @stnguyen90 .

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**